test/jdk/valhalla/valuetypes/LambdaMetaFactory/LambdaTest.java

```diff
@@ -23,62 +23,64 @@
 
 /*
  * @test
+ * @run junit/othervm -XX:+EnableValhalla LambdaTest
```

Comment:
Just style suggestion in the order of jtreg tags, \@summary should come before \@run

Reply:
Hmm... we have a mixture of style.  The jtreg document states `@summary` comes before the action.   I will update them.